content/dstu2/related-person.md

```diff
@@ -31,18 +31,18 @@ Notes:
 
 ### Response
 
-<%= headers 200, GET: '[...]/RelatedPerson?identifier=urn:oid:2.16.840.1.113883.3.13.6|URN:CERNER:IDENTITY-FEDERATION:REALM:AC193C1A-9763-45E1-9FA2-C1C3AA4BA16D:PRINCIPAL:PI98N2FK5TN' %>
+<%= headers 200, GET: '[...]/RelatedPerson?_id=5796348' %>
```

Comment:
Hm I thought that I patched away all the MAX-VALIDATION identifiers but I guess they hang around as old versions

I think it will be understood if we used an example with a total of 40 but only included one or two actual results, replacing the other 38 with one instance of "..." or "<SNIPPED>"